### DISCUSSION TITLE
I want to join your team

### DISCUSSION THREAD

**Discussion by kcraftcode** [GrapesJS/grapesjs#5972] [Ideas]

**Comment by technyk** [comment]
> Beware, this user appears to be a fake account. They have been asking various orgs (including an org I'm in via Discord) for full access without any real reasoning, and their GitHub appears to be bot driven. Some other examples of access requests with little-to-no reasoning:
> 
> 1. [Invite me your team collective.inviting#2](https://github.com/collective/collective.inviting/issues/2)
> 2. [Please add me to the collective #1868](https://github.com/collective/Collective/issues/1868)
> 3. [I want to join this oganization jenkinsci/multibranch-action-triggers-plugin#30](https://github.com/jenkinsci/multibranch-action-triggers-plugin/issues/30)
> 4. [I want to join your team kimurayu45z/ng-algoliasearch#28](https://github.com/kimurayu45z/ng-algoliasearch/issues/28) 
> 
> Most of their repos are forks / fake re-uploads, e.g.https://github.com/kentaurse/blockchain-tutorial which is a clone of https://github.com/nosequeldeebee/blockchain-tutorial but with the git history re-written to make it seem like they authored it.

**Reply by kcraftcode** [reply]
Dear @technyk , I hope this message finds you well. I recently came across a post from you that questions the legitimacy of my account and suggests that I have been making unwarranted access requests. I would like to address these concerns directly. Firstly, I assure you that my account is genuine, and I am a legitimate developer with over 8 years of experience in the industry. I have worked on numerous projects and have a solid track record, as evidenced by my professional portfolio and references. Regarding the accusations about my GitHub repos, while it's true that I have forked and re-uploaded various repositories, this is a common practice among developers to contribute to existing projects, learn from them, and build upon them. I always aim to give credit where it is due, and I apologize if there was any misunderstanding or oversight in the way the history was presented.  If you have any further questions or require additional information, please feel free to contact me directly at [your email]. I request that any false accusations be reconsidered, as they can harm my professional reputation.

**Reply by artf** [reply]
@kentaurse tell us more about blockchains

**Reply by technyk** [reply]
> If you have any further questions or require additional information, please feel free to contact me directly at **[your email]**. I request that any false accusations be reconsidered, as they can harm my professional reputation.

@kentaurse You forgot to fill out your template
(probably from chatGPT eitherway..)

**Reply by artf** [reply]
reported and blocked

**Reply by artf** [reply]
thanks again @technyk for spreading the word about these individuals

**Comment by artf** [comment]
Thanks @technyk looks definitely suspicious
